agent360/agent360.py

```diff
@@ -44,7 +44,7 @@
     from urllib import urlencode
     from urllib2 import urlopen, Request, HTTPError
 
-__version__ = '1.2.34'
+__version__ = '1.2.35'
```

Comment:
I suggest changing the version with a separate commit - this is done by the one who prepares the release.

Reply:
changed version back to 1.2.34